### ISSUE TITLE
File | Open doesn't show initially where am I

### ISSUE THREAD

**Issue by varosi** [leksah/leksah#60]
On Windows 8.1 x64
Leksah 0.14.4.0

When you click on File | Open you don't see in which folder are you:
![leksah](https://cloud.githubusercontent.com/assets/6051718/6501408/96e4db0c-c320-11e4-94b1-42f5e8069277.png)

Only when you choose some folder from left side bar - it is shown on top of the dialog.

**Comment by varosi** [comment]
It's still an issue in 0.15.1.4. Directory field is hidden by default and it is shown only after click in the left pane.

**Comment by hamishmack** [comment]
I think it will also show up if you press /

**Comment by threestrikes** [comment]
Recommend Closed:

Old and solution given.

**Comment by hamishmack** [comment]
This is a quirk of Gtk that we are unlikely to have time to fix.  If someone familiar with windows or perhaps wxHaskell would like to add a more windows like file dialog please feel free to reopen this or add a new issue for that.
